modules/formulize/include/elementdisplay.php

```diff
@@ -220,11 +220,36 @@ function displayElement($formframe="", $ele, $entry="new", $noSave = false, $scr
             $isDisabled = !formulizePermHandler::user_can_edit_entry($form_id, $user_id, $entry);
 		}
 
+		/*

```

Comment:
This block is dealing with permissions, which in the existing file is now handled by line 220. The same comment introduces that condition and this one. This block can be removed.

Reply:
removed the comment block